README.md

```diff
@@ -72,10 +86,13 @@ In this file:
     The argument to the test stimulus block (`c` in this case) is a handle to the module under test.
 5.  In the test body, use `poke`, `step`, and `expect` operations to write the test:
     ```scala
-    c.in.poke(0.U)
-    c.out.expect(0.U)
-    c.in.poke(42.U)
-    c.out.expect(42.U)
+    c.io.in.poke(0.U)
+    c.clock.step()
+    c.io.out.expect(0.U)
+    c.io.in.poke(42.U)
+    c.clock.step()
+    c.io.out.expect(42.U)
+    println("Last output value :" + c.io.out.peek())
```

Comment:
Sorry that I did not catch this last time, but ideally this should be `c.io.out.peek().litValue()` in order to demonstrate to users how to get a scala value.

Reply:
`c.io.out.peek().litValue()` or `c.io.out.peek().litValue` ?